docs/basics/janusgraph-cfg.md

```diff
@@ -52,7 +52,7 @@ General configuration options
 | graph.graphname | This config option is an optional configuration setting that you may supply when opening a graph. The String value you provide will be the name of your graph. If you use the ConfigurationManagement APIs, then you will be able to access your graph by this String representation using the ConfiguredGraphFactory APIs. | String | (no default value) | LOCAL |
 | graph.replace-instance-if-exists | If a JanusGraph instance with the same instance identifier already exists, the usage of this configuration option results in the opening of this graph anwyay. | Boolean | false | LOCAL |
 | graph.set-vertex-id | Whether user provided vertex ids should be enabled and JanusGraph's automatic id allocation be disabled. Useful when operating JanusGraph in concert with another storage system that assigns long ids but disables some of JanusGraph's advanced features which can lead to inconsistent data. EXPERT FEATURE - USE WITH GREAT CARE. | Boolean | false | FIXED |
-| graph.storage-version | The version of JanusGraph storage schema with which this database was created. Automatically set on first start of graph. Should only ever be changed if upgraing to a new major release version of JanusGraph that contains schema changes | String | (no default value) | FIXED |
+| graph.storage-version | The version of JanusGraph storage schema with which this database was created. Automatically set on first start of graph. Should only ever be changed if upgrading to a new major release version of JanusGraph that contains schema changes | String | (no default value) | FIXED |
```

Comment:
This file is generate automatically by executing `mvn --quiet clean install -DskipTests=true -pl janusgraph-doc -am`. You should update related configuration options in Java code and then call `mvn --quiet clean install -DskipTests=true -pl janusgraph-doc -am` instead of changing this file directly. If the result of `mvn --quiet clean install -DskipTests=true -pl janusgraph-doc -am` and `docs/basics/janusgraph-cfg.md` is different, Travis will fail.

Reply:
So, I tried running that (after `export JAVA_HOME=$(java8)`), and it fails on my Darwin Kernel Version 18.7.0: Tue Aug 20 16:57:14 PDT 2019; root:xnu-4903.271.2~2/RELEASE_X86_64 x86_64 system.

Something assumes that `TMPDIR` = `/tmp` which isn't true on macOS.

Reply:
That said, it was helpful, it let me find the one case where the original string was buggily split across two lines.